src/styles/typography/Body/Body2.tsx

```diff
@@ -0,0 +1,29 @@
+import { colors } from "../../colors";
+import { typography } from "../typography";
+
+type Body2Props = {
+    children: React.ReactNode;
+    weight: "regular" | "bold";
+};
+
+
+
+export const Body2 = ({ children, weight }: Body2Props) => {
+    let fontWeight
+    if (weight === "regular") {
+        fontWeight = typography.font.weight.regular
+    } else {
+        fontWeight = typography.font.weight.bold
+    }
+    return (
+        <p
+            style={{
+                fontSize: typography.font.size.xLarge,
+                fontWeight: fontWeight,
```

Comment:
acho que conversamos sobre ternario na nossa call sobre typescript. Nesses casos que é um if e else apenas, poderia usar um ternario aqui:

```suggestion
                fontWeight: typography.font.weight[weight === "regular" ? "regular" : "bold"],
```